datafusion/core/src/physical_plan/memory.rs

```diff
@@ -161,16 +169,13 @@ impl MemoryExec {
             schema,
             projected_schema,
             projection,
-            sort_information: None,
+            sort_information: vec![],
         })
     }
 
     /// Set sort information
-    pub fn with_sort_information(
-        mut self,
-        sort_information: Vec<PhysicalSortExpr>,
-    ) -> Self {
-        self.sort_information = Some(sort_information);
+    pub fn with_sort_information(mut self, sort_information: Vec<LexOrdering>) -> Self {
```

Comment:
Maybe we can add some more information here with a link to somewhere that explains what multiple equivalent orderings are / mean? I think that would make this code easier to use / understand